locales/de/LC_MESSAGES/volto.po

```diff
@@ -2197,9 +2197,9 @@ msgid "Search users…"
 msgstr "Nach Nutzern suchen…"
 
 #: components/manage/Blocks/Search/components/SearchDetails
-# defaultMessage: Searched for
-msgid "Searched for"
-msgstr "Suche nach"
+# defaultMessage: Searched for: <em>{searchedtext}</em>.
```

Comment:
I think it will be fine, the message is formatted like this: https://github.com/plone/volto/blob/e1f789a6e66dddbf554955b9de90686d0028dee1/src/components/manage/Blocks/Search/components/SearchDetails.jsx#L19-L26